src/Umbraco.Web.UI.Client/src/packages/media/media/property-editors/image-cropper/property-editor-ui-image-cropper.element.ts

```diff
@@ -50,10 +67,34 @@ export class UmbPropertyEditorUIImageCropperElement
 		this.dispatchEvent(new UmbChangeEvent());
 	}
 
+	#getValue() {
+		// If the media item is in the recycle bin and media recycle bin protection is on, the media file will have been
+		// renamed to have a .deleted suffix (e.g. media/xxx/test.png will be media/xxx/test.deleted.png on disk).
```

Comment:
This looks a bit dangerous to me, as anyone using the Management API will have to know about this convention. Is there any way we can prevent this from happening client-side?

Reply:
Good point.  And yes, I've found somewhere to do it server-side, so have reverted the client-side amends.